src/index.html

```diff
@@ -11,7 +11,152 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="FieldMain">
+        <legend>Personal information</legend>
+        <div class="field-form">
+          <label for="sur">
+          Surname:
+          </label>
+          <input type="text" name="Surname" 
+          id="sur" autocomplete="off" 
+          required>
+        </div>
+
+        <div class="field-form">
+          <label for="name">
+          Name:
+          </label>
+          <input type="text" name="Name" 
+          id="name" autocomplete="off" 
+          required>
```

Comment:
```suggestion
          <input
            type="text"
            name="Name" 
             id="name"
             autocomplete="off"
             required
           >
```
[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)  point 3

Reply:
Fixed